dataduct/etl/etl_actions.py

```diff
@@ -14,8 +15,10 @@
 logger = logging.getLogger(__name__)
 
 
-URL_TEMPLATE = 'https://console.aws.amazon.com/datapipeline/?#ExecutionDetailsPlace:pipelineId={ID}&show=latest'  # noqa
-
+config = Config()
+REGION = config.etl.get('REGION', None)
+URL_TEMPLATE = 'https://console.aws.amazon.com/datapipeline/?#ExecutionDetailsPlace:pipelineId={ID}&show=latest%s'  # noqa
+URL_TEMPLATE %= 'region=%s' % REGION if REGION is not None else ''
```

Comment:
@tpcstld Oops, AWS Data pipeline is required this url format.

https://console.aws.amazon.com/datapipeline/?region=us-west-2#ExecutionDetailsPlace:pipelineId={ID}&show=latest

If the url is following this, we encounter this error. Actually, I don't know exactly why. ;(

https://console.aws.amazon.com/datapipeline/?#ExecutionDetailsPlace:pipelineId={ID}&show=latest&region=us-west-2

Pipeline with id: xxxx-xxxx does not exist
(Service: DataPipeline; Status Code: 400; Error Code: PipelineNotFoundException;